src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativePerformanceStabilityJudge.kt

```diff
@@ -31,7 +31,8 @@ class RelativePerformanceStabilityJudge {
         return Verdict(
             maxDispersionDifferences.map { (action, maxDispersionDifference) ->
                 val label = action.label
-                val regression = experimentDispersions[label]!! - baselineDispersions[label]!!
+                val regression = extractDuration(experimentDispersions, label, experimentStats.cohort) -
```

Comment:
The multiline, 2x 3-arg function call expression clouds the simplicity of the regression definition.
Let's literraly have the following line of code:
```kotlin
val regression = experimentDispersion - baselineDispersion
```
It's easy to achieve if we extract vals instead of inlining:
```kotlin
val baselineDispersion = extractDuration(baselineDispersions, label, baselineStats.cohort)
```
or even more succinctly:
```
val baselineDispersion = extractDispersion(baselineStats, label)
```